packages/webcert/src/utils/TextWithDynamicLinks.tsx

```diff
@@ -0,0 +1,48 @@
+import { sanitizeText } from '@frontend/common'
+import React from 'react'
+import WCDynamicLink from './WCDynamicLink'
+
+interface Props {
+  text: string
+}
+
+const TextWithDynamicLinks: React.FC<Props> = ({ text }) => {
+  const hasDynamicLink = (text?: string): boolean => {
+    if (!text) {
+      return false
+    }
+    return text.split('<LINK:').length > 1
+  }
+
+  const formatText = (text?: string) => {
```

Comment:
`text` is not optional going in to this component, so no need to have it optional here.